driver/Assembler.ml

```diff
@@ -18,12 +18,17 @@ open Driveraux
 (* From asm to object file *)
 
 let assemble ifile ofile =
-  let cmd = List.concat [
-    Configuration.asm;
+  let cmd,opts = match Configuration.asm with
+  | name::opts -> name,opts
+  | [] -> assert false (* Should be catched in Configuration *) in
+  let opts = List.concat [
+    opts;
     ["-o"; ofile];
     List.rev !assembler_options;
     [ifile]
   ] in
+  let opts = responsefile opts (fun a -> if gnu_system then ["@"^a] else ["@"^a]) in
```

Comment:
Also: I wonder if this logic (of using a response file or not) could be factored out by putting it in `Driveraux.command` or maybe a new function `Driveraux.command_respfile` on top of `Driveraux.command`.